Sources/Casbin/Model/DefaultModel.swift

```diff
@@ -204,13 +204,15 @@ extension DefaultModel:Model {
     public func hasPolicy(sec:String,ptype:String,rule:[String]) -> Bool {
         getPolicy(sec: sec, ptype: ptype).contains(rule)
     }
-    public func getValuesForFieldInPolicy(sec:String,ptype:String,fieldIndex:Int) -> [String] {
-        getPolicy(sec: sec, ptype: ptype).reduce([]) { acc, x in
-            var acc = acc
-            acc.append(x[fieldIndex])
-            return acc
-        }
-    }
+    public func getValuesForFieldInPolicy(sec:String,ptype:String,fieldIndex:Int) -> [String]  {
```

Comment:
`reduce(into:_:)` may be more suitable here.

```swift
let policy = getPolicy(sec: sec, ptype: type).reduce(into: Set<String>()) { acc, x in
    acc.insert(x[fieldIndex])
}
```

Reply:
e, I don't think this is necessary, but thank you.